sdk/communication/azure-communication-phonenumbers/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## 1.2.0 (Unreleased)
 
 ### Features Added
+- Adds env var AZURE_USERAGENT_OVERRIDE, that overrides the header 'x-ms-useragent' on the tests
```

Comment:
nit: "Adds environment variable `AZURE_USERAGENT_OVERRIDE`, that overrides the header `x-ms-useragent` on tests."